src/nvim/ex_docmd.c

```diff
@@ -9239,6 +9239,18 @@ static int ses_do_win(win_T *wp)
   return true;
 }
 
+static int put_view_curpos(FILE *fd, const win_T *wp, char *spaces)
+{
+  int r;
+
+  if (wp->w_curswant == MAXCOL) {
+    r = fprintf(fd, "%snormal! $", spaces);
+  } else {
+    r = fprintf(fd, "%snormal! 0%d|", spaces, wp->w_virtcol + 1);
+  }
+  return r < 0 || put_eol(fd) == FAIL ? FAIL : OK;
```

Comment:
vim-patch:8.1.0333 uses `FALSE`instead of `FAIL` but the return value is compared to `FAIL` in later patch segments.  Wish the expression has brackets to not review C operator precedence again.